cmd/cost/cost.go

```diff
@@ -0,0 +1,95 @@
+package cost
+
+import (
+	"fmt"
+	"github.com/openshift/osd-utils-cli/cmd/common"
+	awsprovider "github.com/openshift/osd-utils-cli/pkg/provider/aws"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"log"
+)
+
+// costCmd represents the cost command
+func NewCmdCost(streams genericclioptions.IOStreams) *cobra.Command {
+	opsCost = newCostOptions(streams)
+	costCmd := &cobra.Command{
+		Use:   "cost",
+		Short: "Cost Management related utilities",
+		Long: `The cost command allows for cost management on the AWS platform (other 
+platforms may be added in the future)`,
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		},
+	}
+
+	//Set flags
+	costCmd.PersistentFlags().StringVarP(&opsCost.accessKeyID, "aws-access-key-id", "a", "", "AWS Access Key ID")
+	costCmd.PersistentFlags().StringVarP(&opsCost.secretAccessKey, "aws-secret-access-key", "x", "", "AWS Secret Access Key")
+	costCmd.PersistentFlags().StringVarP(&opsCost.profile, "aws-profile", "p", "", "specify AWS profile")
+	costCmd.PersistentFlags().StringVarP(&opsCost.configFile, "aws-config", "c", "", "specify AWS config file path")
+	costCmd.PersistentFlags().StringVarP(&opsCost.region, "aws-region", "g", common.DefaultRegion, "specify AWS region")
+
+	//Add commands
+	costCmd.AddCommand(newCmdGet(streams))
+	costCmd.AddCommand(newCmdReconcile(streams))
+	costCmd.AddCommand(newCmdCreate(streams))
+	costCmd.AddCommand(newCmdList(streams))
+
+	return costCmd
+}
+
+var opsCost *costOptions
+
+// costOptions defines the struct for running Cost command
+type costOptions struct {
+	// AWS config
+	accessKeyID     string
+	secretAccessKey string
+	configFile      string
+	profile         string
+	region          string
+
+	genericclioptions.IOStreams
+}
+
+func newCostOptions(streams genericclioptions.IOStreams) *costOptions {
+	return &costOptions{
+		IOStreams: streams,
+	}
+}
+
+//Initiate AWS clients for Organizations and Cost Explorer services using, if given, credentials in flags, else, credentials in the environment
+func (opsCost *costOptions) initAWSClients() (awsprovider.OrganizationsClient, awsprovider.CostExplorerClient, error) {
```

Comment:
Not necessary to have this because there is already https://github.com/openshift/osd-utils-cli/blob/master/pkg/provider/aws/client.go#L66 and https://github.com/openshift/osd-utils-cli/blob/master/pkg/provider/aws/client.go#L103 to init AWS Client.

If organization APIs and costExplorer APIs are added to the Client interface, then we can just use that interface to do all the API calls, don't need to have a separate orgClient and a ceClient.